shrI-sampradAyaH/kriyA/vaikuNTha-dIxita-prapanna-dharma-sAra-samuchchayaH.md

```diff
@@ -32,45 +32,47 @@ TODO: परिष्करणीयम्
 |                                                                               |पुटसङ्ख्यः|
 | :---------------------------------------------------------------------------- | ----: |
 | आचारलक्षणम् | १ |
-| परिभाषा- तत्रादौ शास्त्रनाम निरूपणण्, तत्र वेदाः, उपवेदाः, वेदाज्ञानि, उपनिषगः, त्रगणाः 'स्मृतयः, उपस्मृतयः, पुराणानि, उपपुराणानि, दर्शनानि, संहिताः, आगमशास्त्रम्, शिल्पशास्त्राणि, वैदिकशास्त्र निरूपणम्। | ४ |
-| मोहशा स्त्र निरूपणम् ||
-| शैवपाशुपतादिप्वागमनामधेयम् ||
-| उपागमाळि ||
-| वैदिकशास्त्राकामपि गुणत्रयात्मकत्वं तेषां फलभेदाश्च ओ वैदिकलक्षणम्, परम वैदिक लक्षणम् ||
-| धर्म शास्त्रोपयोगि सञ्ज्ञानिर्वचनम् ||
-| प्रसङ्गाद्व्यतीपा तार्धदयादिलक्षणम् ||
+| परिभाषा- तत्रादौ शास्त्रनाम निरूपणम्, तत्र वेदाः, उपवेदाः, वेदााङ्गानि, उपनिषदः, सूत्रगणाः, स्मृतयः, उपस्मृतयः, पुराणानि, उपपुराणानि, दर्शनानि, संहिताः, आगमशास्त्रम्, शिल्पशास्त्राणि, वैदिकशास्त्र निरूपणम्। | ४ |
+| मोहशास्त्र निरूपणम् ||
+| शैवपाशुपतादिष्वागमनामधेयम् |
+| उपागमाः ||
+| वैदिकशास्त्राणामपि गुणत्रयात्मकत्वं तेषां फलभेदाश्च ||
+| वैदिकलक्षणम्, परमवैदिकलक्षणम् ||
+| धर्मशास्त्रोपयोगि सञ्ज्ञानिर्वचनम् ||
+| प्रसङ्गाद्व्यतीपातार्धोदयादिलक्षणम् ||
 | गुरु-आचार्य ऋत्विगादि लक्षणम्||
-| धर्मप्रशंसा, कलियुगधर्मा :||
-| साषण्णिलक्षणम्, देवलक लक्षणम् ||
-| दीक्षि तादा१ भगवद्भक्तानां देवलकत्वदोसाभावः ||
+| धर्मप्रशंसा, कलियुगधर्माः ||
+| पाषण्णिलक्षणम्, देवलकलक्षणम् ||
+| दीक्षितादां भगवद्भक्तानां देवलकत्वदोषााभावः ||
 | दशविध ब्राह्मण निरूपणम् ||
 | दशविध वैष्णन निरूपणम् ||
 | कर्म ब्रह्म प्रतिपादक शास्त्र विभागः ||
-| कर्मज्ञानादीनाम पै ्यहिकादि फलभेदेन तामसत्वादि निरूपणम्||
+| कर्मज्ञानादीनामप्यैहिकादि फलभेदेन तामसत्वादि निरूपणम्||
 | चक्रादि धारण प्रमाण्य समर्थनम् ||
-| चक्रादि धारण निषेधक वचनाना मुन्यपरत्वनिरूपणम् ||
-| त समुद्राधारण दीक्षा कालः ||
+| चक्रादि धारण निषेधक वचनानामन्यपरत्वनिरूपणम् ||
+| तप्तमुद्राधारण दीक्षाकालः ||
 | मस्त्राभ्यासकालः ||
-| पञ्चनंस्कार प्रयोगारम्भः ||
+| पञ्चसंस्कार प्रयोगारम्भः ||
 | वैष्णवाग्नि करणम् ||
-| ताप ऊर्ध्वपुण्ड्र मस्त्र संस्कारहोमाः ||
+| ताप ऊर्ध्वपुण्ड्र मन्त्र संस्कारहोमाः ||
 | यागहोमः ||
-| पञ्चनंस्कार दी प्रदानम् ||
+| पञ्चनंस्कार दीक्षाप्रदानम् ||
 | शिष्याचार्य क्रमः, तयोर्लक्षणम्, वृत्तिश्च ||
 | वैष्णवेष्टि प्रयोगः ||
-| प्रपत्ति प्रकार निरूपणम्, प्रपन्न पृत्तं प्रपन्नप्पत्तयः। इति परिभाषाप्रकरणम्। ||
+| प्रपत्तिप्रकार निरूपणम्, प्रपन्नवृत्तयः। ||
+ इति परिभाषाप्रकरणम्। 
 | अथाभिगमनम्||
-| धर्मानुष्ठानयोग्य देशाः ||
+| धर्मानुष्ठानयोग्यदेशाः ||
 | नित्यनैमित्तिक कर्मविभागः||
 | प्रातःकाल कृत्य निरूपणारम्भः ||
-| ब्राह्ममुहूर्त क र्तव्यम्, तत्प्रयोगश्च ||
-| प्रातिर्नि र्वेद क्रमः ||
-| योग शेष स्वाध्याय समापनम् ||
+| ब्राह्ममुहूर्त कर्तव्यम्, तत्प्रयोगश्च ||
+| प्रातिर्निर्वेद क्रमः ||
+| योगशेष स्वाध्याय समापनम् ||
 | अभिगमनादि कालपञ्चक निरूपणम् ||
 | अभिगमनादि सङ्कल्पः ||
 | मलमूत्रोत्सर्जनविधिः ||
 | शौचविधिः ||
-| शौचाहमृन्नि रूपणम् ||
+| शौचाहमृन्निरूपणम् |MARKER|
```

Comment:
The original text has this शौचाह्-मृन्निरूपणम् form.  Which form is correct?

Reply:
न भोः। शौचार्ह-मृन्निरूपणम् इत्य् अस्ति खलु। 
![image](https://github.com/vishvAsa/AgamaH_vaiShNavaH/assets/2664797/b902f409-5831-4883-913b-319a24fa30d2)

Reply:
र्ह --> ర్హ  इत्येव लेकनविधिः मया ज्ञातः। 
परिष्कारं करिष्यामि।